src/foam/dao/SQLType.js

```diff
@@ -0,0 +1,64 @@
+foam.ENUM({
```

Comment:
I wouldn't make this an Enum because different databases have different type names and support a long-tail of specialized types, which can still be used by model authors if this is just a String value.

Reply:
Yeah good point. I have already run into issues related to this. PostgreSQL has different type names than say MySQL & SQL Server for things like byte arrays and date times.

Reply:
What would you suggest as an alternative?

Reply:
Just store sqlTypes as Strings. Then you can also make the expressions if they include something like width.

Reply:
Where am I storing those sqlTypes?